fe2-android/app/src/main/res/values/strings.xml

```diff
@@ -294,15 +294,6 @@
   <string name="popcha_scan_title">Enter PoPCHA URL to log in</string>
   <string name="manual_popcha_hint">PoPCHA URL</string>
 
-  <!-- Federation -->
```

Comment:
Where do those deletions come from? Maybe a wrong merge? Those strings look like a recent addition from Federation team, why are they getting deleted in this PR?